jjb/dynamic/scale-ci_baseline.yml

```diff
@@ -0,0 +1,170 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        set -eo pipefail
+
+        # Disable logging
+        set +x
+        oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        # Re-enable logging
+        set -x
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        chmod 600 ${PRIVATE_KEY}
+
+        # Create inventory File:
+        echo "[orchestration]" > inventory
+        echo "${ORCHESTRATION_HOST}" >> inventory
+
+        export ANSIBLE_FORCE_COLOR=true
+        ansible --version
+
+        export WORKLOAD_IMAGE=$WORKLOAD_IMAGE
+        export WORKLOAD_JOB_NODE_SELECTOR=$WORKLOAD_JOB_NODE_SELECTOR
+        export WORKLOAD_JOB_TAINT=$WORKLOAD_JOB_TAINT
+        export WORKLOAD_JOB_PRIVILEGED=$WORKLOAD_JOB_PRIVILEGED
+        export ENABLE_PBENCH_AGENTS=$ENABLE_PBENCH_AGENTS
+        export SCALE_CI_RESULTS_TOKEN=$SCALE_CI_RESULTS_TOKEN
+        export JOB_COMPLETION_POLL_ATTEMPTS=$JOB_COMPLETION_POLL_ATTEMPTS
+        export BASELINE_TEST_PREFIX=$BASELINE_TEST_PREFIX
```

Comment:
Are these export statements required?  Aren't they already env vars in the job due to the parameters?